src/extractor/scripting_environment_lua.cpp

```diff
@@ -186,16 +186,16 @@ void LuaScriptingEnvironment::InitContext(LuaScriptingContext &context)
              // .def(luabind::constructor<>())
              .def_readwrite("forward_speed", &ExtractionWay::forward_speed)
              .def_readwrite("backward_speed", &ExtractionWay::backward_speed)
-             .def_readwrite("name", &ExtractionWay::name)
-             .def_readwrite("ref", &ExtractionWay::ref)
-             .def_readwrite("pronunciation", &ExtractionWay::pronunciation)
-             .def_readwrite("destinations", &ExtractionWay::destinations)
+             .property("name", &ExtractionWay::GetName, &ExtractionWay::SetName)
```

Comment:
if we could pass the `read-write`, we could do without the dedicated setters here.